build.gradle

```diff
@@ -27,7 +28,13 @@ dependencies {
 		force = true
 		because 'CVE-2019-0232 in the transitive tomcat-embed-core dependency'
 	}
-    implementation('org.springframework.data:spring-data-cassandra')
```

Comment:
Do we run the formatter on Groovy files? Some of the tabbing looks inconsistent.

Reply:
I don't think we do... I can go through and try to fix it though